apps/web/src/components/workflows/patient-registration/PatientRegistrationWorkflow.tsx

```
@@ -0,0 +1,1060 @@
+                    id='conditions'
+                    {...form.register('medicalInfo.conditions')}
+                    placeholder='Ex: Diabetes, Hipertensão, etc.'
+                    healthcareContext='patient-registration'
+                    accessibilityAction='input'
+                  />
+                </div>
+              )}
+
+              <div className='space-y-2'>
+                <label className='text-sm font-medium'>Tratamentos estéticos anteriores</label>
+                {medicalInfoFieldArray.fields.map((field, index) => (
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)